main.cpp

```diff
@@ -73,7 +24,22 @@ void testPairToNumber(
     assert(pairNumber == expectedPairNumber);
 }
 
-int main() {
+//new function added that gives all color
+//code with the pair numbers
+
+void printAllCodesAndColors()
+{
+    for (int pairNumber = 1; pairNumber < 26; pairNumber++) {
+        TelCoColorCoder::ColorPair colorPair = TelCoColorCoder::GetColorFromPairNumber(pairNumber);
+        std::cout  << pairNumber << ":  " <<  colorPair.ToString() << std::endl;
```

Comment:
The task was to make a big string with the manual / reference of colors and numbers. Not to output it on the console.